src/Components/IndividualReview.jsx

```diff
@@ -0,0 +1,54 @@
+import { useParams } from "react-router-dom";
+import { fetchReviewById } from "../api";
+import { useEffect, useState } from "react";
+import "./style/IndividualReview.css";
+
+const IndividualReview = () => {
+  const { review_id } = useParams();
+  const [reviewById, setReviewById] = useState({});
+  const [isLoading, setIsLoading] = useState(true);
+  const [date, setDate] = useState("");
+
+  useEffect(() => {
+    fetchReviewById(review_id).then((data) => {
+      setReviewById(data);
+      setIsLoading(false);
+    });
+  }, [review_id]);
+
+  useEffect(() => {
+    const timestamp = reviewById.created_at;
+    const date = new Date(timestamp);
+    const options = { year: "numeric", month: "long", day: "numeric" };
+    const formattedDate = date.toLocaleDateString("en-UK", options);
+
+    setDate(formattedDate);
+  }, [reviewById]);
```

Comment:
this could be extracted out into a normal function, it does not need to be inside a useEffect, it could even be taken out of this component and into a `utils` file and then imported in when it needs using to keep the code looking readable and clean 👍